packages/mongoose/package-lock.json

```diff
@@ -4,42 +4,6 @@
 	"lockfileVersion": 1,
```

Comment:
can we rollback the changes to this file? It shouldn't be affected

Reply:
Done!